diagon-alley/src/GeneralFunctions/FilterOutDetail.js

```diff
@@ -0,0 +1,34 @@
+export const FilterOutDetail = (onlyInStock, fastDelivery, ratingAbove4, newProduct) =>
+        [{
+            fieldLabel:"Exclude out of stock", 
+            fieldForLabel:"excludeOutOfStock",
+            fieldName:"EXCLUDE_OUT_OF_STOCK",
+            fieldType:"checkbox",
+            fieldValue:"excludeOutOfStock",
+            fieldFilter: onlyInStock,
+            fieldLabelClassName:"text-white",
```

Comment:
Don't give extra context. What i mean is you knoe eveything is related to field, then why prefix everything with field?

Reply:
Yes got it will change this